scripts/copydocs.py

```diff
@@ -94,6 +95,13 @@ def parse_args():
         default=True,
         help="Recursively fetch all the referred docs.",
     )
+    parser.add_option(
+        "--editions",
+        dest="editions",
+        action='store_true',
```

Comment:
Let's not mix quotes in new code.
```suggestion
        action="store_true",
```

Reply:
Let's leave the formatting to the bots ; there's a pretty decent cost with having to update a PR, unfortunately -- the time to edit, or the messy history. If we don't care enough to set up automatic formatting rules, then we shouldn't care enough to ask people to _manually_ fix formatting issues.